birdhouse/components/weaver/service-config.json.template

```diff
@@ -1,7 +1,7 @@
 {
     "name": "weaver",
     "url": "https://${PAVICS_FQDN_PUBLIC}/${WEAVER_MANAGER_NAME}",
-    "type": ["ogc-api"],
+    "type": ["service-ogcapi"],
```

Comment:
Need to be specific about *OGC API - Processes*, as there are *Features*, *Maps*, etc.